### PR TITLE
add installation section to eas-cli readme

### PR DESCRIPTION
**Pull request by Simek**
# Why

To improve rideability of `eas-cli` Readme.

# How

This small PR extract the `eas-cli` installation command from the Usage to the separate section, which also includes Yarn command. Additionally I have updated the ToC, and removed the `eas-cli` link which doesn't look that useful.

# Test Plan

GitHub visual markdown editor.

### CODE REVIEW

**Comment by brentvatne** [comment]
the "Usage" section is automatically generated by oclif. we could remove the `<!-- usage -->` tag and associated closing tag to customize that section, but if not the `npm i -g eas-cli` bit will be added back next time the readme is re-generated

**Comment by Simek** [comment]
Sorry, wasn't familiar with `oclif`, but after a bit of research, unfortunately it seems like this line is hardcoded into Readme generator:
* https://github.com/oclif/dev-cli/blob/master/src/commands/readme.ts#L84

According to the `package.json` you are already using a forked version, don't know the reason and if it's a long term, but maybe you can just remove this line and leave the autogenerated "Usage" section?

If it is not possible, I can push a commit removing the `<!-- usage -->` tags, if you are fine with losing this part of autogeneration. Also if any of the solutions won't suits you, feel free to close this PR, this is not that important tweak. 🙂

**Comment by brentvatne** [comment]
@Simek - yeah i think we can remove the usage section entirely, i don't think it's particularly useful.

**Comment by Simek** [comment]
@brentvatne I have removed the autogen tags from TOC and Usage sections. I have also tried to make the Usage more readable, but if you still don't like it or find it not that useful I can just remove it entirely.

File: packages/eas-cli/README.md [hunk 1]
```diff
@@ -23,18 +21,14 @@ yarn global add eas-cli
...
-eas-cli/0.4.3 darwin-x64 node-v14.15.4
-$ eas --help [COMMAND]
-USAGE
-  $ eas COMMAND
-...
+```sh
+eas COMMAND
+# runs the command
+eas (-v|--version|version)
+# prints the version (ex. eas-cli/0.4.3 darwin-x64 node-v14.15.4)
```
Comment (brentvatne) [inline packages/eas-cli/README.md]:
```suggestion
# prints the version
```

File: packages/eas-cli/README.md [hunk 2]
```diff
@@ -7,27 +7,28 @@ EAS command line tool
...
-eas-cli/0.5.0 darwin-x64 node-v14.15.4
-$ eas --help [COMMAND]
-USAGE
-  $ eas COMMAND
-...
+```sh
+eas COMMAND
+# runs the command
+eas (-v|--version|version)
+# prints the version (ex. eas-cli/0.5.0 darwin-x64 node-v14.15.4)
```
Comment (brentvatne) [inline packages/eas-cli/README.md:28]:
```suggestion
# prints the version
```

**Review by brentvatne — APPROVED** [review]

**Pull request merged**